app/components/course/badge_component.html.erb

```diff
@@ -7,7 +7,7 @@
 <% else %>
   <%= link_to path_course_path(course.path, course) do %>
     <div class="shadow-md dark:shadow-none rounded-full flex <%= yass(default_badge: size) %>" data-test-id='default-badge'>
-      <%= image_tag badge, alt: "#{course.title} badge", class: 'p-2.5' %>
+      <%= image_tag badge, alt: "#{course.title}", class: 'p-2.5' %>
```

Comment:
Nit: Within the erb tag we can call the title method directly without wrapping in a string:

```suggestion
      <%= image_tag badge, alt: course.title, class: 'p-2.5' %>
```

Can we do the same with the progress and path badges too please?

Reply:
Sorry I am a beginner at Ruby I took the easy way. 🥲